Self-Consistent-Field/RHF_Gradient.py

```diff
@@ -0,0 +1,176 @@
+"""
+This script calculates nuclear gradients of RHF using
+derivative integrals obtained from PSI4
+
+"""
+
+__authors__ = "Ashutosh Kumar"
+__credits__ = ["Ashutosh Kumar"]
+
+__copyright__ = "(c) 2014-2017, The Psi4NumPy Developers"
+__license__ = "BSD-3-Clause"
+__date__ = "2017-12-17"
+
+import time
+import numpy as np
+np.set_printoptions(precision=15, linewidth=200, suppress=True)
+import psi4
+
+def rhf_gradient(mol, options):
+
+    psi4.core.set_active_molecule(mol)
+    psi4.set_options(options)
+
+    rhf_e, wfn = psi4.energy('SCF', return_wfn=True)
+
+    # Assuming C1 symmetry    
+    occ = wfn.doccpi()[0]
+    nmo = wfn.nmo()
+
+    C = wfn.Ca()
```

Comment:
You can use `Ca_subset("AO", "ALL")` so that it will work even if someone uses symmetry by accident.